packages/components/src/ui/flex/test/__snapshots__/index.js.snap

```diff
@@ -1,312 +1,49 @@
-  --wp-g2-flex-item-margin-bottom: 0;
-  --wp-g2-flex-item-margin-right: calc(4px * 2);
-  --wp-g2-flex-item-margin-right: var(--wp-g2-flex-gap);
-  --wp-g2-flex-item-margin-left: 0;
-  -webkit-align-items: flex-start;
-  -webkit-box-align: flex-start;
-  -ms-flex-align: flex-start;
-  align-items: flex-start;
-  -webkit-flex-direction: row;
-  -ms-flex-direction: row;
-  flex-direction: row;
-  -webkit-box-pack: justify;
-  -webkit-justify-content: space-between;
-  -ms-flex-pack: justify;
-  justify-content: space-between;
-  width: 100%;
-}
-
-@media (prefers-reduced-motion) {
-  .emotion-2.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2 {
-    -webkit-transition: none !important;
-    transition: none !important;
-  }
-}
-
-[data-system-ui-reduced-motion-mode="true"] .emotion-2.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2 {
-  -webkit-transition: none !important;
-  transition: none !important;
-}
-
-.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2 > * + *:not(marquee) {
-  margin-left: calc(4px * 2);
-  margin-left: calc(var(--wp-g2-grid-base) * 2);
-}
-
-.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2.emotion-2 > * {
-  min-width: 0;
-}
+"Snapshot Diff:
+- First value
++ Second value
+
+@@ -7,14 +7,10 @@
+      class=\\"components-flex-item wp-components-flex-item css-uyw6h0\\"
+      data-g2-c16t=\\"true\\"
+      data-g2-component=\\"FlexItem\\"
+    />
+    <div
+-     class=\\"css-1nqjm19\\"
+-   />
+-   <div />
+-   <div
+      class=\\"components-flex-item wp-components-flex-item components-flex-block wp-components-flex-block css-2e3tg1\\"
+      data-g2-c16t=\\"true\\"
+      data-g2-component=\\"FlexBlock\\"
+    />
+  </div>"
+`;
 
-<div
-  class="components-flex wp-components-flex emotion-2"
-  data-g2-c16t="true"
-  data-g2-component="Flex"
->
-  <div
-    class="components-flex-item wp-components-flex-item emotion-0"
-    data-g2-c16t="true"
-    data-g2-component="FlexItem"
-  />
-  <div
-    class="components-flex-item wp-components-flex-item components-flex-block wp-components-flex-block emotion-1"
-    data-g2-c16t="true"
-    data-g2-component="FlexBlock"
-  />
-</div>
+exports[`props should render align 1`] = `
+"Snapshot Diff:
+- First value
++ Second value
+
+- Snapshot Diff:
+- - First value
+- + Second value
+-
+- @@ -1,11 +1,11 @@
+-   Object {
+-     \\"--wp-g2-flex-gap\\": \\"calc(var(--wp-g2-grid-base) * 2)\\",
+-     \\"--wp-g2-flex-item-margin-bottom\\": \\"0\\",
+-     \\"--wp-g2-flex-item-margin-left\\": \\"0\\",
+-     \\"--wp-g2-flex-item-margin-right\\": \\"var(--wp-g2-flex-gap)\\",
+- -   \\"align-items\\": \\"flex-start\\",
+- +   \\"align-items\\": \\"center\\",
+-     \\"box-sizing\\": \\"border-box\\",
+-     \\"display\\": \\"flex\\",
+-     \\"flex-direction\\": \\"row\\",
+-     \\"font-family\\": \\"var(--wp-g2-font-family)\\",
+-     \\"font-weight\\": \\"var(--wp-g2-font-weight)\\","
```

Comment:
Here is an example of `toMatchStyleDiffSnapshot`. You'll notice that the diff notes that `align-items` has changed between the two values.